client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-view/motion-view.component.ts

```diff
@@ -413,7 +438,7 @@ export class MotionViewComponent extends BaseMeetingComponent implements OnInit,
 
     private addOriginMotionTab(id: Id): void {
         const originMotion = this.repo.getViewModelUnsafe(id);
-        if (!this.originMotionsLoaded.find(m => m.id === id)) {
+        if (!this.originMotionsLoaded.find(m => m.id === id) && originMotion) {
```

Comment:
```suggestion
        if (!this.originMotionsLoaded.find(m => m.id === id)) {
```

Either check for `originMotion` first or remove it. I don't think this check is necessary.

Reply:
If I don't check for this then the tab is not visible at all
I'll move it in front